app/editor/src/features/admin/topic-score-rules/validation/TopicScoreRuleSchema.ts

```diff
@@ -8,10 +8,10 @@ export const TopicScoreRuleSchema = yup.object().shape({
   pageMax: yup
     .string()
     .optional()
-    .when('pageMin', (pageMin: string, schema: any) => {
+    .when('pageMin', (pageMin: string[], schema: any) => {
```

Comment:
Why would this be an array of pageMin values?

Reply:
For the new version it seems the ConditionBuilder type would only accept an array instead of an object as the first parameter, otherwise, an error will be thrown.

Reply:
Looking now!

Reply:
Looks like for whatever reason they decided to go with an array in the new version
From the docs:
`Schema.when(keys: string | string[], builder: object | (values: any[], schema) => Schema): Schema`
@Fosol